chalicelib/checks/audit_checks.py

```diff
@@ -662,15 +662,21 @@ def check_opf_status_mismatch(connection, **kwargs):
                               for fileset in exp['other_processed_files']
                               for item in fileset['files'] if fileset['title'] == title])
             statuses = set([opf_status_dict[f['uuid']] for f in file_list])
-            if len(statuses) > 1:  # status mismatch in opf collection
-                problem_dict[title] = {f['@id']: {'status': opf_status_dict[f['uuid']]} for f in file_list}
-                if hg_dict.get(title):
-                    problem_dict[title][hg_dict[title]] = {'status': opf_status_dict[hg_dict[title]]}
+            # import pdb; pdb.set_trace()
+            if not statuses:
+                # to account for empty sections that may not yet contain files
+                pass
+            elif len(statuses) > 1:  # status mismatch in opf collection
+                scores = set([STATUS_LEVEL.get(status, 0) for status in list(statuses)])
+                if len(scores) > 1:
+                    problem_dict[title] = {f['@id']: {'status': opf_status_dict[f['uuid']]} for f in file_list}
+                    if hg_dict.get(title):
+                        problem_dict[title][hg_dict[title]] = {'status': opf_status_dict[hg_dict[title]]}
             elif hg_dict.get(title) and STATUS_LEVEL[list(statuses)[0]] != STATUS_LEVEL[opf_status_dict[hg_dict[title]]]:
                 if not (list(statuses)[0] == 'pre-release' and opf_status_dict[hg_dict[title]] == 'released to lab'):
                     problem_dict[title] = {'files': list(statuses)[0],
                                            'higlass_view_config': opf_status_dict[hg_dict[title]]}
-            elif 'release' not in result['status'] and (STATUS_LEVEL[result['status']] < STATUS_LEVEL[list(statuses)[0]]):
+            elif 'released' not in result['status'] and (STATUS_LEVEL[result['status']] < STATUS_LEVEL[list(statuses)[0]]):
```

Comment:
If the ExpSet is "released to project" and opfs are "released", this will not be caught, right?

Reply:
Good catch.  I entirely removed the 'released' bit and currently getting same result on data.  Maybe something will show up for a submission that is in progress that will remind us why that was there but I think it should work.